nilearn/connectome/connectivity_matrices.py

```diff
@@ -231,6 +231,52 @@ def sym_to_vec(symmetric, discard_diagonal=False):
     return symmetric[..., tril_mask] / scaling[tril_mask]
 
 
+def vec_to_sym(vec):
```

Comment:
The name is a bit hard to understand out of context. How about "vec_to_sym_matrix". A bit longer, but much more explicit.